Tetris Clone C++/TetrisPiece.hpp

```diff
@@ -33,6 +33,7 @@ class TetrisPiece
     int x, y;
     // col, row are grid-based positions
     int col, row;
+    int * frameCounter;
```

Comment:
can we do that then? maybe have the main loop query it to check if the piece should drop or maybe have the framecounter be public or something